mobile/build.gradle

```diff
@@ -102,6 +102,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'androidx.multidex:multidex:2.0.1'
+    implementation 'android.arch.work:work-runtime:1.0.0-alpha12'
```

Comment:
There's a 1.0.0-beta02 now, but we can also leave that up to dependabot and/or update once it's final.

Reply:
Let the bot do that :)